bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -261,8 +261,16 @@ class Base extends Component {
     if ((loading || !subscriptionsReady) && !meetingHasEnded && meetingExist) {
       return (<LoadingScreen>{loading}</LoadingScreen>);
     }
-
-    if (ejected) {
+    
+    if (( meetingHasEnded || ejected || userRemoved ) && meetingIsBreakout) {
+      Base.setExitReason('breakoutEnded').finally(() => {
+        Meteor.disconnect();
+        window.close();
+      });
+      return null;
+    }
+    
+    if (ejected && !meetingIsBreakout) {
```

Comment:
```suggestion
    if (ejected) {
```

It's unnecessary to validate `meetingIsBreakout` once breakout users will always go into the previous condition.